src/Sulu/Bundle/AdminBundle/DependencyInjection/Configuration.php

```diff
@@ -47,10 +47,10 @@ public function getConfigTreeBuilder()
             ->children()
                 ->arrayNode('resources')
                     ->useAttributeAsKey('resourceKey')
-                    ->arrayPrototype()
+                    ->prototype('array')
                         ->children()
                             ->arrayNode('form')
-                                ->scalarPrototype()->end()
+                                ->prototype('scalar')->end()
```

Comment:
same as for arrayPrototype the scalarPrototype only exists since 3.3 prototype('scalar') need to be used instead: http://symfony.com/doc/3.3/components/config/definition.html#array-nodes